specification/cluster/stats/types.ts

```diff
@@ -152,6 +152,12 @@ export class SearchUsageStats {
   rescorers: Dictionary<Name, long>
   sections: Dictionary<Name, long>
   retrievers: Dictionary<Name, long>
+  /* @availability stack since=9.2.0 */
+  extended: Dictionary<Name, ExtendedSearchUsage>
```

Comment:
This can also be {} - is there a better way to represent this?

Reply:
Those results look expected to me. 

Unfortunately the `retrievers.text_similarity_reranker.chunk_rescorer` is the only field that we are sending in for 9.2. However, we anticipate that going forward, we'll definitely have other retrievers supported, with their own key:value pairs, and probably other calls for example for `search` or `rescorers`. Furthermore, when I implemented this @Mikep86 specifically asked for more flexibility so that more nesting could be returned in the future should we decide it was necessary. Therefore, we're kind of stuck with maps.

Reply:
Yeah, I was thinking of keeping it as a long until we need to change it - we don't know if/when/what might change in the future so long seemed easier. What's preferable?

Reply:
OK then let's be as granular as possible to avoid breaking changes. Updated in 39e38a1cc1f8d9a57ad505ec44ad408129f3f243